openbb_terminal/econometrics/econometrics_controller.py

```diff
@@ -458,7 +458,15 @@ def call_export(self, other_args: List[str]):
             type=str,
         )
 
-        parser.add_argument(
+        export_group = parser.add_mutually_exclusive_group(required=True)
+        export_group.add_argument(
+            "-f",
```

Comment:
The `--help`, `-h` flags do not work now.

The way that this works across the rest of the Terminal is: `command --export filename.xlsx`, or `command --export xlsx`.  The main difference here is that the dataset must be selected as a choice, and the command name is called `export`.  There should, however, only be one argument which covers both situations, with a way to write to a specific `--sheet-name`.

```
(🦋) /econometrics/ $ export --help

usage: export [-n NAME] (-f FILE | -t {xlsx,csv}) [-h]
export: error: one of the arguments -f/--file -t/--type is required


(🦋) /econometrics/ $ export -h

usage: export [-n NAME] (-f FILE | -t {xlsx,csv}) [-h]
export: error: one of the arguments -f/--file -t/--type is required
```

Reply:
Fixed.